tox.ini

```diff
@@ -73,13 +73,13 @@ passenv =
 # Can be used to specify which tests to run, e.g.: tox -- -s
 commands =
     python -c "import os; os.system('install-custom-pkgs --install-boto-wheels')"
-    pip install 'apache-airflow==2.6.3' --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.6.3/constraints-3.7.txt"
+    pip install 'apache-airflow==2.7.0' --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.7.0/constraints-3.8.txt"
 
```

Comment:
Setting constraint to 3.10 causes error in py38 unit test like below. Since apache-airflow for python_version <3.9 requires importlib-metadata<5.0.0 and >=1.7 but constraints-3.10.txt specifies importlib-metadata==6.8.0

```


ERROR: Cannot install apache-airflow==2.7.0 because these package versions have conflicting dependencies.
--
 
The conflict is caused by:
apache-airflow 2.7.0 depends on importlib-metadata<5.0.0 and >=1.7; python_version < "3.9"
The user requested (constraint) importlib-metadata==6.8.0
 
To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
 
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
WARNING: You are using pip version 21.3; however, version 23.2.1 is available.
You should consider upgrading via the '/codebuild/output/src1906118588/src/.tox/py38/bin/python -m pip install --upgrade pip' command.
ERROR: InvocationError for command /codebuild/output/src1906118588/src/.tox/py38/bin/pip install apache-airflow==2.7.0 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.7.0/constraints-3.10.txt (exited with code 1)

```